dev/devicelab/lib/framework/runner.dart

```diff
@@ -125,3 +127,33 @@ Future<VMIsolateRef> _connectToRunnerIsolate(int vmServicePort) async {
     }
   }
 }
+
+Future<void> cleanupSystem() async {
+  print('\n\nCleaning up system after task...');
+  final String javaHome = await findJavaHome();
+  if (javaHome != null) {
+    print('\nTelling Gradle to shut down (JAVA_HOME=$javaHome)');
+    final String gradlewBinaryName = Platform.isWindows ? 'gradlew.bat' : 'gradlew';
+    final Directory tempDir = Directory.systemTemp.createTempSync('flutter_devicelab_shutdown_gradle.');
+    recursiveCopy(new Directory(path.join(flutterDirectory.path, 'bin', 'cache', 'artifacts', 'gradle_wrapper')), tempDir);
+    copy(new File(path.join(path.join(flutterDirectory.path, 'packages', 'flutter_tools'), 'templates', 'create', 'android.tmpl', 'gradle', 'wrapper', 'gradle-wrapper.properties')), new Directory(path.join(tempDir.path, 'gradle', 'wrapper')));
```

Comment:
Might be worth a comment explaining why we have to do this (I guess we can't run `gradlew` from an arbitrary folder that doesn't have a properties file?)

Reply:
will do